src/components/button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {ButtonStyled, ButtonText} from './styles/button';
+import {FieldContainer} from './styles/general';
+
+interface buttonProps {
+  content: string;
+  pressButon: () => void;
+}
+
+export const StyledButton: React.FC<buttonProps> = props => {
```

Comment:
```suggestion
interface buttonProps {
  content: string;
  pressButon: () => void;
}

export const StyledButton = (props: StyledButtonProps) => {
```

Aqui é mais prouqe o tipo React.FC parece que não é mais muito útil no React 18.